### ISSUE TITLE
HBO max and Netflix videos not playing

### ISSUE THREAD

**Issue by CrateTech** [nfriedly/node-unblocker#185]
Hello, I know that this has probably been asked before but when I go to HBO Max or Netflix and click play on a video, I either get an error for HBO Max or nothing happens like on Netflix. I had just accepted that this is just the way it was, but I was messing around with Taco Proxy V2 (https://github.com/Tacosheel/TacoProxyV2) and they can play HBO Max. The only problem is that I cannot use that proxy because it does not support YouTube. So I was wondering if there was any way to play HBO Max or Netflix.

**Comment by nfriedly** [comment]
I think you might have to just have two proxies then. It's cool that they got it to work, but I don't think I'm going to work in this one anytime soon.

**Comment by CrateTech** [comment]
👍 I am making this proxy for my coworkers as well so I will try and build them into the same webpage.

**Comment by yodaluca23** [comment]
Maybe this will work?

https://github.com/ab77/netflix-proxy 

If @nfriedly could somehow Incorporate it into nodeunblocker??

**Comment by nfriedly** [comment]
No, that wouldn't work. That's a DNS proxy, not a web proxy. The web content still comes directly from a netflix server, just one that's in a different region than you'd normally connect to. (Because they make different movies available in different regions.)
